spec/support/features/session_helper.rb

```diff
@@ -162,7 +162,7 @@ def acknowledge_and_confirm_recovery_code
       click_on button_text, class: 'recovery-code-continue'
 
       code_words.size.times do |index|
-        fill_in "recovery-#{index}", with: code_words[index].downcase
+        fill_in "recovery-#{index}", with: code_words[index].downcase + 'xxxx'
```

Comment:
the new `maxlength` attribute on the input means that the extra characters are ignored, which is what we want and why the tests continue to pass.